pkg/urbit/jets/e/lore.c

```diff
@@ -3,6 +3,30 @@
 */
 #include "all.h"
 
+  static u3_noun
+  _lore_cut(c3_w pos_w, c3_w len_w, u3_noun src)
+  {
+    if ( 0 == len_w ) {
+      return 0;
+    }
+    else {
+      // as an optimization, we alloc small lines on the stack
+      // (only small to prevent stack overflow)
+      c3_t  big_t = len_w > 255;
+      c3_y* buf_y = big_t ? u3a_malloc(len_w) : alloca(len_w);
```

Comment:
this is the perfect chance to introduce a "technical" reason for the 80 column rule!